design/k8s-job-hook.md

```diff
@@ -150,27 +111,65 @@ hooks:
 Each item list will match the `registry/name:tag` used in the hook.
 If the tag is omitted, any tag will be allowed.
 
+
 ### Execution
 
-lakeFS requests job creation from the cluster.  In case the request fails it will log the job information and fail the hook execution.
-By default the hook waits for the job execution to complete, and based on the job status consider the execution to be successful.
-It will capture the job output into the step output.
-In case we would not like to block the operation (merge/commit) until the job execution is completed, for example on post-* events. We can change the default behavior by setting wait_for_complete to false:
+lakeFS requests job creation from the cluster.  Job information will be captured to the job's log.
+In case the request fails it will log the error and fail the hook execution.
+lakeFS will consider job creation as successful execution. Will log the job information without waiting for the job to complete or capturing the output.
 
-```yaml
-properties:
-  wait_for_complete: false
+### Authorizations
+
+Base on the above, lakeFS will require the following permissions:
+
+- `job` get, create and watch
+- `pod` get
+- `pod/log` get, list, watch
+
+The following describes possible ClusteRole that enables the above. We can limit the scope to a single namespace using Role.
```

Comment:
But you used `lakefs-hook` namespace above, so maybe use a role here to be aligned with the above example?